src/data/faq.json

```diff
@@ -1,2819 +1,2893 @@
-                            "<ul><li>The signal attenuation must have been below 79dB for at least 5 minutes.</li>",
-                            "<li>The transmission risk level must be at least III (3). It is determined when sharing the diagnosis codes per day according to defined criteria such as symptom status and/or symptom onset.</li></ul>",
-                            "If these criteria are <b>not</b> met, an encounter is <b>not counted</b> within the app. It is a 'non-risk encounter'."
-                        ]
-                    },
-                    {
-                        "title": "How does the app calculate the risk (version >=1.9)?",
-                        "anchor": "encounter_19_calc",
-                        "active": false,
-                        "textblock": [
-                            "With the introduction of Corona-Warn-App version 1.9, a new version of the Exposure Notification Framework (ENF) from Apple/Google is used, version 2.0 (also known as ExposureWindow mode by Google). The data from this interface is used as the basis for calculating the individual risk.",
-                            "While the interface's first version 1.0 used average attenuation values for all encounters with another device during the course of the day, the new version 2.0 now considers 30-minute time windows.",
-                            "Each of these windows must meet certain criteria to be considered:",
-                            "<ul><li>The signal attenuation must have been below 79dB for at least 5 minutes within the 30-minute window</li>",
-                            "<li>The transmission risk level must be at least III (3). It is determined when sharing the diagnosis codes per day according to defined criteria such as symptom status and/or symptom onset</li></ul>",
-                            "If these criteria are <b>not</b> met, an encounter is <b>not counted</b> within the app. It is a 'non-risk encounter'. These encounters will still show up in the operating system's contact log, but do not count towards the overall risk score in the app.",
-                            "If the conditions are met, the contact times are weighted depending on the signal attenuation:",
-                            "<ul><li>Times with an attenuation <63dB are weighted at 80%.</li>",
-                            "<li>Times with an attenuation >= 63dB and <73dB are weighted at 100%.</li>",
-                            "<li>Times with an attenuation >= 73dB and <79dB are weighted at 10%.</li></ul>",
-                            "Based on the transmission risk level (III to VIII), a factor between 0.6 and 1.6 is determined. The <i>weighted</i> exposure time from the previous step is multiplied by this factor, which then results in the <i>normalized</i> exposure time. Finally, all normalized times of a day are added up. Ultimately, this means that several short low risk encou
```

Comment:
"risk map" =>

"risk card"

"map" is an incorrect translation of "Karte" in German